tests/dummy/app/view-components/nrg-forms/template.hbs

```diff
@@ -69,7 +69,7 @@
         <field.checkbox @label="Inline Checkbox" @model={{this}} @valuePath="checkbox1" @disabled={{disabled}} />
       </form.field>
       <form.field @required={{required}} as |field|>
-        <field.checkbox @type="slider" @label="Inline Slider" @checked={{true}} @disabled={{disabled}} />
+        <field.checkbox @type="slider" @label="Inline Slider" @disabled={{disabled}} />
```

Comment:
Does this need a model and valuePath?

Reply:
If I wanted to do anything important with it, yes.
In this case it's only there to demonstrate the style variations of the checkbox, so it's value is unbound.